research-hub-web/src/app/components/about/about.component.ts

```diff
@@ -8,6 +8,9 @@ import { ResearchHubApiService } from '@services/research-hub-api.service';
   styleUrls: ['./about.component.scss']
 })
 export class AboutComponent {
+  public title: String = "About";
+  public summary: String = "The Centre for eResearch comprises a team of highly qualified research and technical staff dedicated to the delivery of advanced computational solutions to help power the University's research mission.";
+  public coverImageUrl: String = 'url(../../../assets/images/banner15.jpg)';
```

Comment:
- [ ] As we're going to be using our local images often, add a new `assets` typescript path to `tsconfig.base.json`